platform/retarget.cpp

```diff
@@ -177,6 +192,14 @@ extern "C" FILEHANDLE PREFIX(_open)(const char* name, int openmode) {
         init_serial();
         return 2;
     }
+#if defined MBED_CFG_DEBUG_OPTIONS_COVERAGE
+    else if(coverage_report) {
```

Comment:
Nit: The style guide says we always use spaces after statements like `if`: https://github.com/ARMmbed/mbed-os/blob/master/docs/COMMITTERS.md#rules